build/build-image.sh

```diff
@@ -41,14 +41,37 @@ dir=$1
 image=$2
 
 docker build "$ROOT" \
-    -f $dir/Dockerfile \
-    -t cortexlabs/${image} \
-    -t cortexlabs/${image}:${CORTEX_VERSION}
+  -f $dir/Dockerfile \
+  -t cortexlabs/${image} \
+  -t cortexlabs/${image}:${CORTEX_VERSION}
 
 if [ "$slim" == "true" ]; then
-    docker build "$ROOT" \
+  docker build "$ROOT" \
+    -f $dir/Dockerfile \
+    --build-arg SLIM=true \
+    -t cortexlabs/${image}-slim \
+    -t cortexlabs/${image}-slim:${CORTEX_VERSION}
+fi
+
+if [ "${image}" == "python-predictor-gpu" ]; then
+  cuda=("10.0" "10.1" "10.2" "11.0")
+  cudnn=("7" "7" "7" "8")
+
+  # the only tag for the fat version is for CUDA 10.1 and CUDNN 7
+  docker build "$ROOT" \
+    -f $dir/Dockerfile \
+    --build-arg CUDA_VERSION=${cuda[1]} \
+    --build-arg CUDNN=${cudnn[1]} \
+    -t cortexlabs/${image}:${CORTEX_VERSION}-cuda${cuda[1]}
```

Comment:
I'm thinking maybe this section is not necessary, since this image will already have been built above, and perhaps we don't need the cuda tag on the fat version (since there is only one). What do you think? We can discuss it live too if you prefer